### ISSUE TITLE
"Add filename as title" setting not having any effect

### ISSUE THREAD

**Issue by hannahildebrandt** [KosmosisDire/obsidian-webpage-export#260]
**Describe the bug**
When "Add filename as title" is selected, documents without a header don't automatically generate one based on the filename.

**Operating System**
Windows 11

The only errors that appear in the console are a series of "failed to find render leaf", but these appear only after the documents have been processed.

My understanding of this setting is that it should make the documents appear like they do in Obsidian, with the name of the document appearing as an H1 header above the document. However, when I export using this setting, documents that lack H1 headers at the top of their body still appear with no header at all.

(As an aside, it would be nice to have the option for this to ALWAYS be true regardless of whether or not the document starts with an H1, so you can match the existing formatting used in Obsidian. I have a lot of documents that use H1 tags for organization, but I'd still like to have the filename appear at the top.)

**Comment by KosmosisDire** [comment]
Hi, thanks this is a duplicate of #234 and #132, so I will go ahead and close this in favor of those. #234 should be fixed in the next update

**Comment by hannahildebrandt** [comment]
Sorry for wasting your time! I had thought I had searched for this, and it turns out I had a totally different repo open.

**Comment by KosmosisDire** [comment]
Absolutely no worries! ☺️
